Documentation/running-exporters.md

```diff
@@ -0,0 +1,66 @@
+# Running Exporters
+Running exporters and scraping them with prometheus-operator.
+
+## Two approaches to utilizing ServiceMonitors
+What approach is the "best" is currently mostly a "personal" preference.
+
+### One ServiceMonitor for multiple Services
+You create a ServiceMonitor to target a label that is shared among the Services you want to monitor.
+This approach enables you to use less ServiceMonitors, but may require as mentioned above a shared label among all services to be monitored.
+
+### One ServiceMonitor per one Service
+You create a ServiceMonitor to target only the labels of the **one** specific Service you want to target.
+This approach enables you to see the targeted pods separated in these groups, but requires more ServiceMonitors to be created.
+
+## Create a service that exposes Pod(s)
+For scraping an exporter or metrics "interface" you need a service that targets the pod(s) of the exporter or application.
```

Comment:
> `metrics "interface"`

I've never heard of them being referred as that. Do you have a reference? If you intended to describe it as that then maybe restructure the sentence a bit to not confuse people and they start searching on google for that.